packages/autorest.typescript/test/rlcIntegration/generated/lroRest/package.json

```diff
@@ -27,8 +27,8 @@
     "@azure/core-rest-pipeline": "^1.5.0",
     "@azure/logger": "^1.0.0",
     "tslib": "^2.6.2",
-    "@azure/core-lro": "^2.5.4",
-    "@azure/abort-controller": "^1.0.0"
+    "@azure/core-lro": "3.0.0-beta.1",
```

Comment:
when do we plan to GA core-lro?

Reply:
i plan to GA core lro in May and the pr is here https://github.com/Azure/azure-sdk-for-js/pull/29450.